apps/ledger-live-mobile/scripts/e2e-ci.mjs

```diff
@@ -15,6 +15,8 @@ const usage = (exitCode = 1) => {
 };
 
 const build_ios = async () => {
+  await $`pnpm mobile exec detox clean-framework-cache`;
```

Comment:
What's the impact of cleaning and rebuilding the the framework cache every time ? Are the detox tests going to take longer to execute ?

Reply:
As we can see [here](https://github.com/LedgerHQ/ledger-live/actions/runs/11280727365/job/31374345686) it only take 3 seconds to do it so it should not impact to much.
 
<img width="876" alt="Screenshot 2024-10-14 at 10 45 47" src="https://github.com/user-attachments/assets/e51a783c-53f0-4fe6-9d32-d79e71584438">

Reply:
Alright, good to know. Thanks !